src/router.js

```diff
@@ -9,11 +9,33 @@ export default router;
 // eslint-disable-next-line no-unused-vars
 const upload = multer({ dest: service.UPLOADS_FOLDER });
 
+
+//for main page
 router.get("/", async (req, res) => {
-	let games = await service.getGames();
-	res.render("index", { games });
+	const page = Math.max(parseInt(req.query.page) || 1, 1);
```

Comment:
note that if `parseInt(req.query.page) = 0`, `0 || 1 = 1`, i guess the output is gonna be fine anyways, but i feel something like
```suggestion
	const parsed = parseInt(req.query.page);
	const page = Number.isNaN(parsed) ? 1 : Math.max(parseInt(req.query.page), 1);
```
is more robust somehow